kubo/020-pin-cids.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/env sh
+
+set -eu
```

Comment:
How long it takes? What if Kubo already stores this cid?

As I understand correctly, failing of this script prevents Kubo to startup?

Reply:
> How long it takes?

The time to download CIDs depends on the count of files, network conditions, etc. In my tests it worked fast though.

> What if Kubo already stores this cid?

It will just move to the next one in the list.

> failing of this script prevents Kubo to startup

That's right. I think we can move the content of the script to a separate file and wrap a call to it to prevent kubo from failure on startup.